x-pack/plugin/sql/qa/src/main/java/org/elasticsearch/xpack/sql/qa/jdbc/DataLoader.java

```diff
@@ -198,6 +195,9 @@ private static void loadEmpDatasetIntoEs(RestClient client, String index, String
                     }
                     hadLastItem = true;
                     bulk.append('"').append(titles.get(f)).append("\":\"").append(fields.get(f)).append('"');
+                    if (titles.get(f).equals("gender") && extraFields) {
+                        bulk.append(",\"extra_gender\":\"M\"");
```

Comment:
Personally I would prefer it to be in tha employees.csv file and have a different value from `gender` like `Male` and `Female`.

Reply:
Imo, it's a too impactful change to do it like this.

Reply:
Apologies, I mixed up the feature with an enum keyword that can take a set of predefined values in the mapping. It's ok as is, but maybe check if `null` is also acceptable.

Reply:
But maybe having is a `Male` instead of `M` can help avoid confusion in the future regarding tests?